hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -218,11 +218,25 @@ def process(configfile='salt://hubblestack_pulsar/hubblestack_pulsar_win_config.
                             if leftover.startswith(os.sep):
                                 _append = False
         if _append and config_found:
+            r['Hash'] = _get_item_hash(r['Object Name'], config.get('checksum', 'sha256'))
             new_ret.append(r)
     ret = new_ret
 
     return ret
 
+def canary(change_file=None):
```

Comment:
lost in commit https://github.com/hubblestack/hubble/commit/7cccf7b9e92dfbc32c0253bc3d63418962852dcd